### PR TITLE
void

### PR DESCRIPTION
**Pull request by abebeos**

### CODE REVIEW

**Comment by gmarkall** [comment]
Thanks very much for the PR, I've queued it for review.

File: buildscripts/incremental/setup_conda_environment.sh [hunk 1]
```diff
@@ -63,9 +63,12 @@ if [ "${VANILLA_INSTALL}" != "yes" ]; then
 fi
 
 # Install the compiler toolchain
+arch=`uname -m`
 if [[ $(uname) == Linux ]]; then
     if [[ "$CONDA_SUBDIR" == "linux-32" || "$BITS32" == "yes" ]] ; then
         $CONDA_INSTALL gcc_linux-32 gxx_linux-32
+    elif [[ "$arch" == "ppc64le" ]] ; then
+        $CONDA_INSTALL gcc_linux-"$arch" gxx_linux-"$arch"
```
Comment (sklam) [inline buildscripts/incremental/setup_conda_environment.sh:71]:
```suggestion
        # This branch is not exercised by Numba's Azure CI for the main fork.
        $CONDA_INSTALL gcc_linux-"$arch" gxx_linux-"$arch"
```

**Review by sklam — COMMENTED** [review]
We don't usually accept code that is not exercised anywhere but we are making this an one-off exception since we are currently unable to add more CI services to exercise other architectures.

File: buildscripts/incremental/setup_conda_environment.sh (hunk 1 above)
Comment (abebeos) [reply buildscripts/incremental/setup_conda_environment.sh:71]: Don't understand this, to be honest, but pressed the button anyways.

**Pull request closed without merging**